packages/gatsby-plugin-catch-links/src/gatsby-browser.js

```diff
@@ -2,8 +2,8 @@ import { navigate } from "gatsby"
 
 import catchLinks from "./catch-links"
 
-exports.onClientEntry = () => {
-  catchLinks(window, href => {
```

Comment:
@pieh Would you consider a signature change to this function a breaking change? Not really used by anyone besides this file but just wondering 🤔

Reply:
Probably not an issue since this file isn't meant to be a consumed by others and is not a public API

Reply:
No - this is internals change - `catchLinks` function is not documented as public API